ramls/loan-policy-storage.raml

```diff
@@ -39,6 +41,7 @@ resourceTypes:
         501:
           description: "Not implemented yet"
     post:
+      is: [validate]
```

Comment:
This changes the API definition, which needs to be reflected in a version change. I'm not sure whether this should be considered a breaking compatible change or not.